.github/workflows/python.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4.2.1
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v5.2.0
+        uses: actions/setup-python@v5.3.0
```

Comment:
Can we just change all uses of `actions/setup-python@...` to `actions/setup-python@v5`.

Then we don't need to review every little change to that action.